package.json

```diff
@@ -24,7 +25,6 @@
     "@types/node-fetch": "^1.6.9",
     "@types/prettyjson": "^0.0.29",
     "@types/request": "^2.48.5",
-    "apollo-boost": "^0.4.9",
```

Comment:
The Apollo Boost project is now retired https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#apollo-boost:~:text=The%20Apollo%20Boost%20project%20is%20now%20retired